include/lsst/meas/modelfit/CModel.h

```diff
@@ -305,12 +301,15 @@ struct CModelControl {
 
     CModelControl() :
         psfName("DoubleGaussian"),
-        minInitialRadius(0.1)
+        minInitialRadius(3E-3),
+        fallbackInitialMomentsPsfFactor(1.5)
     {
         initial.nComponents = 3; // use very rough model in initial fit
         initial.optimizer.gradientThreshold = 1E-2; // with coarse convergence criteria
         initial.optimizer.minTrustRadiusThreshold = 1E-2;
         dev.profileName = "luv";
+        exp.nComponents = 6;
+        exp.optimizer.maxOuterIterations = 250;
     }
```

Comment:
Good catch.  I'd never actually synced up these parameters across the two codebases, since they're implemented differently.  I'll add a commit to do that before merging.